File: MT.Model/Business/ProjectExperiment.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectExperiment:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectExperiment:EntityBase.
    /// </summary>
    public class ProjectExperiment : EntityBase
    {
        [Key]
        public Guid ProjectExperimentID { get; set; }
        public Guid ProjectID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }

        public Guid FileResourceID { get; set; }
    }
    #endregion
}

File: BECore/app/model/ProjectExperiment.js
/// <summary>
/// Model EXTJS ProjectExperiment.
/// </summary>
Ext.define('QLDT.model.ProjectExperiment', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectExperimentID',
    idType: 'Guid',
    fields: [
        { name: "ProjectExperimentID", type: "string" },
		{ name: "ProjectID", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

